### PR TITLE
fix: add cspell ignore for CTRV

### PR DESCRIPTION
**Pull request by h-ohta**
## Description

add cspell ignore for CTRV to pass spell-check-partial

## Tests performed

pass spell-check-partial

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: perception/multi_object_tracker/CMakeLists.txt [hunk 1]
```diff
@@ -26,6 +26,7 @@ set(MULTI_OBJECT_TRACKER_SRC
   src/data_association/mu_successive_shortest_path/mu_successive_shortest_path_wrapper.cpp
   src/tracker/motion_model/motion_model_base.cpp
   src/tracker/motion_model/bicycle_motion_model.cpp
+  # cspell: ignore ctrv
```
Comment (h-ohta) [inline perception/multi_object_tracker/CMakeLists.txt]:
```suggestion
  # cspell:ignore ctrv
```

File: perception/multi_object_tracker/CMakeLists.txt [hunk 2]
```diff
@@ -26,6 +26,7 @@ set(MULTI_OBJECT_TRACKER_SRC
   src/data_association/mu_successive_shortest_path/mu_successive_shortest_path_wrapper.cpp
   src/tracker/motion_model/motion_model_base.cpp
   src/tracker/motion_model/bicycle_motion_model.cpp
+  # cspell:ignore ctrv
```
Comment (h-ohta) [inline perception/multi_object_tracker/CMakeLists.txt:29]:
```suggestion
  # cspell: ignore ctrv
```

File: perception/multi_object_tracker/include/multi_object_tracker/tracker/model/pedestrian_tracker.hpp [hunk 3]
```diff
@@ -24,6 +24,8 @@
 
 #include <kalman_filter/kalman_filter.hpp>
 
+// cspell:ignore CTRV
```
Comment (h-ohta) [inline perception/multi_object_tracker/include/multi_object_tracker/tracker/model/pedestrian_tracker.hpp:27]:
```suggestion
// cspell: ignore CTRV
```

File: perception/multi_object_tracker/include/multi_object_tracker/tracker/motion_model/ctrv_motion_model.hpp [hunk 4]
```diff
@@ -32,6 +32,8 @@
 #endif
 #include <geometry_msgs/msg/twist.hpp>
 
+// cspell:ignore CTRV
```
Comment (h-ohta) [inline perception/multi_object_tracker/include/multi_object_tracker/tracker/motion_model/ctrv_motion_model.hpp:35]:
```suggestion
// cspell: ignore CTRV
```

File: perception/multi_object_tracker/src/tracker/motion_model/bicycle_motion_model.cpp [hunk 5]
```diff
@@ -28,6 +28,7 @@
 #include <Eigen/Core>
 #include <Eigen/Geometry>
 
+// cspell:ignore CTRV
```
Comment (h-ohta) [inline perception/multi_object_tracker/src/tracker/motion_model/bicycle_motion_model.cpp:31]:
```suggestion
// cspell: ignore CTRV
```

**Review by HansRobo — APPROVED** [review]
LGTM from the spell check setting point of view

**Review by YoshiRi — APPROVED** [review]
Thanks! LGTM

**Pull request merged**